tests/boss/test_initialrebalancer.cpp

```diff
@@ -53,17 +53,15 @@ class DummyEarningsManager {
  */
```

Comment:
Maybe this PR is a good way to remind us that listpeerchannels is now a way to iterate over all the channels?